src/t8_forest/t8_forest_vtk.cxx

```diff
@@ -127,13 +130,219 @@ typedef int         (*t8_forest_vtk_cell_data_kernel) (t8_forest_t forest,
                                                        void **data,
                                                        T8_VTK_KERNEL_MODUS
                                                        modus);
+                                                       
+/* lookup table for number of nodes for curved eclasses. */
+const int           t8_curved_eclass_num_nodes[T8_ECLASS_COUNT] =
+  { 1, 3, 8, 6, 20, 10, 15, 13 };
+
+/* lookup table for vtk types of curved elements */
+const int           t8_curved_eclass_vtk_type[T8_ECLASS_COUNT] =
+  { 1, 21, 23, 22, 25, 24, 26, 27 };
+
+/* 
+ * depending on whether we want to write curved or non-curved elements
+ * we need the right number of points, so we choose the right lookup table
+ */
```

Comment:
This and the function below must be protected by VTK macros.
Otherwise we get an unused function warning when not compiling with VTK